llvm/test/tools/sycl-post-link/spec-constants/SYCL-2020-zeroinitializer-array-of-arrays.ll

```diff
@@ -111,6 +112,40 @@ attributes #2 = { convergent "frame-pointer"="all" "no-trapping-math"="true" "st
 ; CHECK-PROP-NEXT: 9f47062a80eecfa7____ZL8coeff_id=2|gNAAAAAAAAAAAAAAAAAAAQAAAAQAAAAAEAAAAQAAAAgAAAAAIAAAAQAAAAwAAAAAMAAAAQAAAAABAAAAQAAAAQAAAAQBAAAAUAAAAQAAAAgBAAAAYAAAAQAAAAwBAAAAcAAAAQAAAAACAAAAgAAAAQAAAAA
```

Comment:
I think that we can now consider dropping those base64-encoded strings and only check property key (and maybe its type `=2`), because the content can now be checked through the pass debug option